content/docs/tinacms-context.md

```diff
@@ -74,11 +74,22 @@ export default App
 
 Instead of having the full `tinacms` code in your production site, your main production bundle will just contain the much smaller `tinacms/dist/edit-state` bundle (>2kb).
 
-## Manually toggling edit-mode
+## Toggling edit-mode
 
-You can enter and exit edit mode by tapping into the `useEditState` hook. A common pattern is to place this hook on an "admin" page, which simply puts you into edit mode and sends you back to the page you were on.
+You can log into edit mode by visiting `/admin` and log out of edit mode by visiting `/admin/logout`.
```

Comment:
I wanted to use the same language on the pages themselves here:  "Login" and "Logout" rather than "Enter" and "Exit".